VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -12,4 +12,5 @@
     android:singleLine="true"
     android:textColor="@color/edx_grayscale_neutral_dark"
     android:textSize="@dimen/edx_x_small"
+    android:textDirection="locale"
```

Comment:
OK
> On 12-Sep-2015, at 1:52 am, Brian Guertin <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/edx_spinner_item.xml <https://github.com/edx/edx-app-android/pull/359#discussion_r39316046>:
> 
> > @@ -12,4 +12,5 @@
> >      android:singleLine="true"
> >      android:textColor="@color/edx_grayscale_neutral_dark"
> >      android:textSize="@dimen/edx_x_small"
> > +    android:textDirection="locale"
> Oh, and in this case it's actually a layout issue; Spinners have an arrow on one side and text on the other. Without setting this to locale, the text is aligned beside the arrow, rather than opposite it.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/359/files#r39316046>.
>